core/lexicon/en/topmenu.inc.php

```diff
@@ -17,7 +17,7 @@
 $_lang['api_docs_desc'] = 'Complete API documentation for MODX Revolution.';
 $_lang['bespoke_manager'] = 'Manager Customization';
 $_lang['bespoke_manager_desc'] = 'Manage bespoke MODX Managers';
-$_lang['components'] = 'Extras';
+$_lang['components'] = 'Packages';
```

Comment:
No. Extras are what could be installed by a package, but they are not the same thing.

Reply:
Why so complicate? We can leave only "Packages" and "Components" - installed packages.
Why else are the entities "Extras" and "Add-ons"?

Reply:
Add-Ons/Extras/Components <-- these are interchangeable and different than packages. I'm fine with picking one of those, but replacing any of those with packages is simply wrong. It's not complicated at all.

Reply:
The more important issue is these things need to be discussed independently from all the miscellaneous changes included in this single PR.

Reply:
Issue hang in 2017, for example, https://github.com/modxcms/revolution/issues/13554 but there is no progress in the discussion.